FirebaseSessions/Sources/Public/SessionsDependencies.swift

```diff
@@ -15,18 +15,62 @@
 
 import Foundation
 
-// Sessions Dependencies determines when a dependent SDK is
-// installed in the app. The Sessions SDK uses this to figure
-// out which dependencies to wait for to getting the data
-// collection state.
-//
-// This is important because the Sessions SDK starts up before
-// dependent SDKs
+private final class AtomicBox<T> {
```

Comment:
This should probably be moved into Core; we can eventually make it into our backported `OSAllocatedUnfairLock` implementation.

Reply:
Agreed, thanks.  I'll look into refactoring it out into Core or SharedSwift in a subsequent PR.